sass/style-ec.scss

```
@@ -18,7 +18,11 @@
 @use "./components/toolbar";
 @use "./components/wysiwyg";
 @use "./components/global";
-@use "./components/description_list";
+@use "./components/description_list" with (
+  $background-color: var(--ecl-color-neutral-40),
+  $border-color: var(--ecl-color-neutral-40),
+);
+
```

No need to override the variable values since you're setting the default ones.
```suggestion
@use "./components/description_list";
```